tests/modules/js-and-css/embed-optimizer/embed-optimizer-test.php

```diff
@@ -11,9 +11,11 @@ class Embed_Optimizer_Helper_Tests extends WP_UnitTestCase {
 	/**
 	 * Test that the oEmbed HTML is filtered.
 	 *
-	 * @dataProvider test_filter_oembed_html_data
+	 * @covers ::embed_optimizer_filter_oembed_html
+	 * @covers ::embed_optimizer_lazy_load_scripts
```

Comment:
This one `@covers` was added erroneously. I've removed it in a sub-PR: https://github.com/WordPress/performance/pull/984